### ISSUE TITLE
mitotic spindle elongation (again)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12431]
Sorry David ! I think we can improve this.
GO:0000022 mitotic spindle elongation
The cell cycle process in which the distance is lengthened between poles of the mitotic spindle. Mitotic spindle elongation begins during mitotic prophase and ends during mitotic anaphase B.

This paper is useful: PMID:9802907 (I should have done this one earlier)

Abstract
In higher eukaryotic cells, the spindle forms along with chromosome condensation in mitotic prophase. In metaphase, chromosomes are aligned on the spindle with sister kinetochores facing toward the opposite poles. In anaphase A, sister chromatids separate from each other without spindle extension, whereas spindle elongation takes place during anaphase B. 

Intro
early mitotic to late anaphase events were observed in living fission yeast cells. S. pombe has three phases in spindle dynamics, spindle formation (phase 1), constant spindle length (phase 2), and spindle extension (phase 3). Sister centromere separation (anaphase A) rapidly occurred at the end of phase 2. The centromere showed dynamic movements throughout phase 2 as it moved back and forth and was transiently split in two before its separation
During the early stages of mitosis, the SPBs separate from each other, the spindle forms between them, and cy- toplasmic microtubules disappear (e.g., Masuda et al., 1992). The interphase centromeres are known to be clustered and located near the SPB (Takahashi et al., 1992; Funabiki et al., 1993). In situ hybridization with the centromere DNA indicated that this interphase centromere clustering was disrupted on entry into mitosis, and that the individual centromeres appeared to interact with the short spindle (Funabiki et al., 1993; Saitoh et al., 1997).

Phase 1 corresponded to the period of spindle formation (e.g., a prophase-like stage), in which the spindle length in- creased from 0 to 2.5 􏰈m; phase 2 represented the period of constant spindle length (e.g., metaphase–anaphase A); and phase 3 was the period of spindle extension from 2.5 to 12–15 􏰈m (anaphase B). These three distinct phases were found in all the wild-type cells examined, although the duration of each phase was strongly dependent on the temperature used (Figure 2, B–E).

So what this says is  the "elongation" which occurs during metaphase is actually "(mitotic) spindle assembly"
and the  "rapid elongation" that occurs in anaphase B is "(mitotic) spindle extension"

This seems to hold from yeast to human, so I wonder if we should disambiguate like this  and have "(mitotic) spindle  elongation as a broad synonym for both terms?

We need to separate because the 2  processes are quite different and at present there is a danger of lumping the annotation....

**Comment by mah11** [comment]
The spindle can get longer (defined as pole-to-pole distance) either microtubule polymerization or by midzone MTs sliding along each other to push the poles apart (thereby making the area of overlapping, interdigitated polar MTs smaller). So, questions:

1) Is MT polymerization always "spindle assembly"? If not, where does assembly stop and elongation of an existing spindle by MT polymerization begin? This may depend on whether any MT polymerization takes place in "phase 3".

2) Is the requested "spindle extension" identical to the MT-based movement that pushes the poles apart? 

I don't know the answers, but it looks key to defining the requested terms.

Incidentally, I don't think using the word "extension" adds any distinction - in plain English it actually has a somewhat broader meaning than "elongation". If MT polymerization is always assembly in the spindle context, then we can use "elongation" for the movement-based part; if there is polymerization-mediated elongation, then we'll need long "elongation by x" terms. (Sorry.)

**Comment by ValWood** [comment]
1) Is MT polymerization always "spindle assembly"? If not, where does assembly stop and elongation of an existing spindle by MT polymerization begin? This may depend on whether any MT polymerization takes place in "phase 3".

Not sure...I think both but I'd need to check 

2) Is the requested "spindle extension" identical to the MT-based movement that pushes the poles apart?

GO:1905121
Name	microtubule sliding involved in mitotic spindle elongation

In anaphase B, yes (but could also be polymerization happening).
I also don't know if any sliding occurs  during the premetaphase part. I previously assumed not as this is when the midzone (where the sliding originates) is formed....

I need to  do more of the papers on my list. If I don't find out I'll ask someone.....

(PS, I  presume extension means elongating the existing structure, so this seemed a useful way to differentiate...perhaps not..)

**Comment by ukemi** [comment]
Looking at the ontology and I think we have it right. I think that we need to distinguish what we observe to happen from the underlying mechanism. The parent term describes the process we observe, mitotic spindle elongation exactly as it is defined. The children should describe the mechanisms that result in the poles getting further apart. To me it seems that this is accomplished in two ways, polymerization and sliding. What you are trying to disambiguate is when each of these mechanisms are at play. Am I missing something?

**Comment by ukemi** [comment]
Looking closer, it looks like e are missing the polymerization component of the mechanism. We have microtubule bundle formation, which is a negative regulation of depolymerization, but we don't have a polymerization term.

**Comment by ValWood** [comment]
Its more about the 2 different stages. They really are quite different. The first part (in metaphase) is when the spindle assembles. The second extension is after  the chromosomes separate, and its rapid extension. Different gene products involved (at least, its different genes which cause defects at both time points).

Mostly people seem refer to the first phase as assembly , and the second phase (or phase 3 in the paper above) as elongation. However this isn't always the case (which is why we now have the term referring to extension during metaphase and anaphase), which is why I think it warrants clarification.

**Comment by mah11** [comment]
Elongation means making an existing structure longer. Extension = elongation for anything where only one dimension is relevant; it isn't any more specific.

**Comment by ValWood** [comment]
Ah in that case it seems odd to refer to the metaphase  part as 'elongation' at all. No?

**Comment by mah11** [comment]
If the spindle isn't getting longer (pole to pole), then no, no "spindle elongation" occurs. I guess "extension" could maybe occur in another dimension, but I don't think I've heard of that actually happening in metaphase.

**Comment by ValWood** [comment]
I guess it hinges on when the spindle is considered to be "formed" because it cannot extend until it is assembled, which is basically what you say above.
My inclination is:
assembles/forms in metaphase 
period of stable length
extends in anaphase B

**Comment by mah11** [comment]
Doesn't spindle assembly begin in prophase?

**Comment by ValWood** [comment]
yep fixed

**Comment by ukemi** [comment]
Val,

What should I do with this ticket? It looks as though GO:0051256 mitotic spindle midzone assembly should be removed as a child of elongation and the gloss in the definition should be changed to "Mitotic spindle elongation happens during mitotic anaphase. Will this suffice?

**Comment by ValWood** [comment]
Hi David. I'll take this ticket, and check with some experts, then pass it back to you.

**Comment by ValWood** [comment]
confirmation of these elongation stages, for mitosis, meiosis I & II...

intro, Repositioning of Aurora B Promoted
by Chiasmata Ensures Sister Chromatid Mono-Orientation in Meiosis I
Takeshi Sakuno,1,2 Koichi Tanaka,1 Silke Hauf,3 and Yoshinori Watanabe1,*
PMID: 21920317

In wild-type cells, the elongation of the spindle occurs in three distinct steps at both meiosis I and meiosis II, as in mitosis (Nabeshima et al., 1998; Yamamoto et al., 2008); a short spindle elongates (phase I), stays at a constant length (phase II), and then further elongates (phase III) (Figure 1A, WT). Phase II and phase III roughly corre- spond to metaphase and anaphase, respectively.

**Comment by ValWood** [comment]
I think the suggestion will work, can ;look at this next week?
v

**Comment by ukemi** [comment]
Yes. Let's look next week.

**Comment by ValWood** [comment]
- [ ] mitotic spindle assembly does not have mitotic spindle organization as a parent
- [ ] mitotic spindle organization does not have "GO:1902850 microtubule cytoskeleton organization involved in mitosis" as a parent
probably should?

**Comment by ValWood** [comment]
based on "a short spindle elongates (phase I), stays at a constant length (phase II), and then further elongates (phase III) (Figure 1A, WT). Phase II and phase III roughly correspond to metaphase and anaphase, respectively."

does 
"mitotic spindle assembly" correspond to the "elongation" which occurs in phase 1?

**Comment by ValWood** [comment]
Decided to add
- [ ]  mitotic spindle elongation during mitotic prophase
- [ ] mitotic spindle elongation during mitotic anaphase
